src/org/zaproxy/zap/extension/pscanrulesBeta/ServletParameterPollutionScanner.java

```diff
@@ -112,6 +107,12 @@ public void scanHttpResponseReceive(HttpMessage msg, int id, Source source) {
 		}
 	}
 
+	private boolean isValidActionAttribute(Attribute att) {
+		boolean isActionAttribute = att.getName().equalsIgnoreCase("action");
```

Comment:
It could return immediately, no need to do further checks if it's not an action.